starter/js/atm.js

```diff
@@ -1,15 +1,53 @@
 //Begin with the document ready function
+$( document ).ready(function() {
+});
+
+
+// START BELOW
+
+
+
+    var checkingBalance = 0;
+    var savingsBalance = 0;
 
     //Checking account deposit function
+    var depositChecking = function() {
+      var amount = parseInt(document.querySelector('#amountChecking').value);
+      document.querySelector('#amountChecking').value = "";
+
+      checkingBalance += amount;
 
+      document.querySelector('#checkingBalance').innerHTML = "$" + checkingBalance;
+
+      if (checkingBalance === 0) {
+        document.querySelector('#checkingBalance').className = "balance red";
```

Comment:
You'll need to do this check every time – in the withdraw also. So maybe you'll want to extract it into a function:
```js
function checkingColor(balance){
  if (balance === 0) {
    document.querySelector('#checkingBalance').className = "balance red";
  }
  else {
    document.querySelector('#checkingBalance').className = "balance";
  }
}
```
And then you can call it by running `checkingColor(checkingBalance);`. Same pattern for the savings account. (You could make it DRYer by actually writing one function that would check the color for both checking *and* savings, but that would be more complex!)